pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -686,8 +718,17 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 	if err != nil {
 		return err
 	}
-	nodeCfg.DNSBindAddress = ""
-	nodeCfg.DNSIP = ""
+	version, err := h.serverVersion()
+	if err != nil {
+		return err
+	}
+	if useBindAddress(version) {
+		nodeCfg.DNSBindAddress = "127.0.0.1:53"
+		nodeCfg.DNSIP = "172.30.0.1"
```

Comment:
this is really highlighting how problematic the "don't embed an IP" code is.  The dns ip should point to the bind address, which then ensures that the code in the node dns is used.  That is the long term flow and avoids having to query etcd for endpoints.  If we aren't going to use that, we shouldn't start it (don't pass bind address).  What happened when only dns ip was set to the service ip?